src/test/datascience/kernel-launcher/kernelFinder.vscode.test.node.ts

```diff
@@ -118,11 +118,11 @@ suite('DataScience - Kernels Finder', () => {
         ) as LocalKernelConnectionMetadata;
         assert.ok(juliaKernelSpec);
 
-        const kernelSpec = await kernelFinder.findKernel(resourceToUse, {
+        const kernelSpec = (await kernelFinder.findKernel(resourceToUse, {
             kernelspec: juliaKernelSpec?.kernelSpec as any,
             orig_nbformat: 4
-        });
-        assert.ok(kernelSpec);
-        assert.deepEqual(kernelSpec, juliaKernelSpec);
```

Comment:
Curious, why we're not comparing the kernel specs anymore.
Do you know what's different? Because previously we were looking for an exact match and now the match has been relaxed.

Reply:
I'd prefer to have an exact match, after all, both should return the same object. Finding the kernel and listing should return the same thing

Reply:
Before we only searched local kernels. Now we search both. So either one can come out.

Reply:
The test code above was wrong. It 'finds' the expected julia kernel just based on the language. The kernelFinder.findKernel is more specific. It's searching through all kernels to find one and when we're doing remote it finds the remote one first.